internal/security/secretstore/init.go

```diff
@@ -53,11 +53,13 @@ import (
 const (
 	addKnownSecretsEnv   = "ADD_KNOWN_SECRETS"
 	redisSecretName      = "redisdb"
+	mqttSecretName       = "mqtt-bus"
```

Comment:
Thinking we may want to be more generic like:
```suggestion
	messageBusSecretName       = "message-bus"
```

Reply:
So is it the case that all other messageBus secret are mutually exclusive? i.e. there is no concurrently using message bus secret types?

Reply:
eg. mqtt bus + nats?

Reply:
> So is it the case that all other messageBus secret are mutually exclusive? i.e. there is no concurrently using message bus secret types?

Correct no mixing of MessageBus implementations. That wouldn't work.